From 394ac1c39a564efb1db982334abb8f10e34b8e26 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Sat, 20 Sep 2025 23:17:25 +0200
Subject: [PATCH] Bigger password limits

Requires password to have at least 8 characters, but allow up to 256
characters.
---
 js/src/password/Password.js | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/js/src/password/Password.js b/js/src/password/Password.js
index b9f03b58..4c9ca853 100644
--- a/js/src/password/Password.js
+++ b/js/src/password/Password.js
@@ -212,11 +212,11 @@ function validator(formData) {
 function validatePassword(password) {
     if (password === "") return _("Choose a password.");
 
-    if (password.length < 6)
-        return _("Password should have at least six symbols.");
+    if (password.length < 8)
+        return _("Password should have at least eight symbols.");
 
-    if (password.length > 128)
-        return _("Password cannot have more than 128 symbols.");
+    if (password.length > 256)
+        return _("Password cannot have more than 256 symbols.");
 
     return null;
 }
-- 
2.51.0

